.travis.yml

```diff
@@ -25,8 +30,10 @@ before_script:
   - if [ $OLDEST_SHARED == $OLDEST_COMMIT ]; then git fetch --unshallow; fi
 
 stages:
-  - prepare
-  - test
+  - name: prepare
+  - name: test
+  - name: release
+    if: ((branch = master) OR (branch =~ /^\d+\.\d+\.x$/)) AND (type = push) AND (NOT fork)
```

Comment:
I'm assuming this is the correct equals here. It's a syntax diversity meeting here

Reply:
I think it's right but we shall see, got this from https://docs.travis-ci.com/user/conditions-v1